torch/_utils.py

```diff
@@ -126,20 +146,57 @@ def _unflatten_tensors(flat, tensors):
     return tuple(outputs)
 
 
+def _unflatten_sparse_tensors(flat, tensors):
+    """View flat buffer (containing indices and values) using the sizes of
+    tensors. Assume that tensors are of same sparse type, and that flat is given
+    by _flatten_sparse_tensors.
```

Comment:
I understand this is a helper function, so the naming/docs are not as important (i.e., "read the source code if you actually want to know what it does"). However, for future reference when writing docstrings, I'd recommend adding: (1) the precise types of all the arguments (e.g., flat is a tuple of (list of dense tensor, list of dense tensor)) and tensor is a list of sparse tensors; (2) the type returned (list of sparse tensors).

Reply:
Thanks Ed. I will add those. :)